src/platform/testing/unit/mocha-setup.js

```diff
@@ -8,6 +8,7 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { JSDOM } from 'jsdom';
 import 'blob-polyfill';
+import 'babel-polyfill';
 // import sinon from 'sinon'
```

Comment:
We could also move this into `platform/testing/unit/mocha.opts` as -
```
--require babel-polyfill
```
Not sure which way makes more sense.

Reply:
Yeah, it probably makes sense to keep all the side-effect-only imports in the mocha.opts file. I'll move this and the `blob-polyfill` up (assuming all the unit tests pass locally when I do so).